deploy-agent/deployd/common/utils.py

```diff
@@ -123,10 +124,16 @@ def run_prereqs(config):
 
 def get_info_from_facter(keys):
```

Comment:
consider using a decorator for all this, it might not be possible everywhere

Reply:
discussed - might revisit this idea in a future diff

Reply:
discussed - we might revisit this pattern in a future diff